src/index.ts

```diff
@@ -89,7 +89,15 @@ export default class Plugin extends AbstractBtpPlugin {
   constructor (opts: IlpPluginMiniAccountsConstructorOptions,
     { log, store }: IlpPluginMiniAccountsConstructorModules = {}) {
     super({})
-    this._port = opts.port || 3000
+    if (opts.wsOpts && opts.wsOpts.port && opts.port) {
+      throw new Error('Specify one of `opts.wsOpts.port` or `opts.port` but not both. Or specify neither to use the default port number.')
```

Comment:
This is pretty verbose. Maybe: ``"Specify at most one of: `ops.wsOpts.port`, `opts.port`"``.

Reply:
Good suggestion!